extensions/adobe/experience/adcloud/campaigndetails.schema.json

```diff
@@ -0,0 +1,67 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id": "https://ns.adobe.com/experience/adcloud/campaigndetails",
+  "title": "Ad Campaign Detail",
+  "type": "object",
+  "description": "Adobe AdCloud Ad Campaign Hierarchy Details",
+  "definitions": {
+    "campaignDetails": {
+      "properties": {
+        "https://ns.adobe.com/experience/adcloud/campaigndetails/accountId": {
+          "title": "Account ID",
+          "type": "integer",
```

Comment:
should this really be an integer?

Reply:
Currently used as string however changed to string for versatility.